tools/untranslated.xml

```diff
@@ -4,25 +4,24 @@
     <!-- iOS specific untranslated strings -->
     <string name="hide">Hide</string>
     <string name="login_error_title">Unable to login!</string>
-    <string name="start_chat">Start Chat</string>
-    <string name="dismiss">Dismiss</string>
+        <string name="dismiss">Dismiss</string>
     <string name="photo">Photo</string>
-    <string name="import_contacts">Import Device Contacts</string>
-    <string name="import_contacts_message">To chat with contacts from your device open the settings menu and enable the Contacts option</string>
-    <string name="synchronizing_account">Synchronizing Account</string>
-    <string name="invalid_qr_code">Invalid group QR Code</string>
-    <string name="no_account_setup">No Account set up</string>
-    <string name="delete_account">Delete Account</string>
-    <string name="delete_account_message">Are you sure you wante to delete your account data?</string>
+    <string name="import_contacts">Import device contacts</string>
+    <string name="import_contacts_message">To chat with contacts from your device open the settings menu and enable the Contacts option.</string>
+    <string name="synchronizing_account">Synchronizing account</string>
+    <string name="invalid_qr_code">Invalid group QR code.</string>
+    <string name="no_account_setup">No account set up.</string>
+    <string name="delete_account">Delete account</string>
+    <string name="delete_account_message">Are you sure you want to delete your account data?</string>
     <string name="danger">Danger</string>
     <string name="next">Next</string>
     <string name="reset">Reset</string>
     <string name="edit_contact">Edit contact</string>
-    <string name="custom_port">Custom Port</string>
+    <string name="custom_port">Custom port</string>
     <string name="automatic">Automatic</string>
-    <string name="complete">Complete</string>
+    <string name="completed">Completed</string>
     <string name="info">Info</string>
-    <string name="qr_code_title">QR scan/show</string>
+    <string name="qr_code_title">Scan/Show QR code</string>
```

Comment:
`Scan/Show QR code` This string might be too long for the tab bar where it is used